specification/msi/ManagedIdentity.Management/Identity.tsp

```diff
@@ -0,0 +1,94 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using Azure.ClientGenerator.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.ManagedIdentity;
+/**
+ * Describes an identity resource.
+ */
+model Identity
+  is Azure.ResourceManager.TrackedResource<UserAssignedIdentityProperties> {
+  ...ResourceNameParameter<
+    Resource = Identity,
+    KeyName = "resourceName",
+    SegmentName = "userAssignedIdentities",
+    NamePattern = ""
+  >;
+}
+@@visibility(Identity.properties, Lifecycle.Read);
```

Comment:
Is this by converter or by hand? Original Swagger's "readOnly" shouldn't have any effect:
https://github.com/Azure/azure-rest-api-specs/blob/84f223e8bea800a1a0b503b8c28675abf91f92a2/specification/msi/resource-manager/Microsoft.ManagedIdentity/stable/2024-11-30/ManagedIdentity.json#L541-L546

<img width="768" height="492" alt="Screenshot 2025-08-14 at 12 04 42" src="https://github.com/user-attachments/assets/b8321f3b-eebe-43f5-bb70-460b8660121c" />

Reply:
@wanlwanl As discussed, we should remove the `@visibility` on `Identity` and `IdentityUpdate`.

/cc @weidongxu-microsoft @pshao25

Reply:
removed